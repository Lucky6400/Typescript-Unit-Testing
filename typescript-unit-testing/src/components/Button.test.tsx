import { render } from '@testing-library/react'
import Button from './Button'

test('renders correctly and matches the snapshot', () => {
   const { asFragment } = render(<Button label='MyButton' />);

   expect(asFragment()).toMatchSnapshot('Button with "Click me" label');
})