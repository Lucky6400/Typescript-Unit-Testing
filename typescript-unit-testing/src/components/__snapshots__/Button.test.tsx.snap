// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot: Button with "Click me" label 1`] = `
<DocumentFragment>
  <button>
    MyButton
  </button>
</DocumentFragment>
`;
